src/GettingStarted/Introduction.mdx

```diff
@@ -0,0 +1,111 @@
+
+### Why this is important?
+
+Creating UIs from scratch can be hard, but it doesn’t have to be. Blue Design System is here to guide you throughout the design and development processes to make composing new interactions simple. Our mantra has always been, "Let us solve the easy problems so that you can focus on the hard ones."
+
+### Philosophy
+
+The underpinnings of the Blue Design System and the way we create components are founded upon four core principles:
+
+#### Accessibility
+
+All of our components are made for everyone. They meet the latest accessibility standards, as we strive to build a unified experience for both visual display and screen reader users. In each component’s documentation, we’ve included an accessibility guide on what to pay attention to when implementing a component and on what accessibility features are included in the component itself.
+
+#### Flexibility
+
+All of our components are built with flexibility in mind. Since UIs are meant to be composed of multiple elements, the components we produce are meant to be used in harmony. We take in to account the diverse array of use cases that we can anticipate for everything we build so that our components can fit anywhere in an experience.
+
+If a UI is a symphony, each component can be thought of as an instrument meant to fill its role so that together they create a beautiful concerto.
+
+#### Easy to use
+
+All of our components are simple and easy to use. Every component has a well-documented implementation guide full of examples, advanced scenarios, and notes for things to keep in mind when using them. Some components have an interactive playground to experiment with the components to learn more about them.
+
+Additionally, if a component has a javascript need, there is a well documented component API with information on how to interact with that component and use its interface.
+
+#### Responsiveness
+
+All of our components are responsive on all screen sizes. As we continue to look towards the future, it’s important that each of our components can be used on any device. This means we don’t have desktop components and mobile components – we have components that are optimized for everywhere.
+
+#### Methodology
+
+We strive to make our components as flexible and easy to use as possible. On every component page we’ve included documentation for all our user personas (designer, developers, and non-techies) as well as guidelines on how and when to use each of the components.
+
+#### Implementation
+
+Although the UI is composed of multiple elements, you should never combine components – nest them instead.
+
+```
+<div class="bh-avatar">
+    <div class="bh-badge"></div>
+</div>
+```
+
+Notice how the badge in the example above is nested inside of the avatar and not combined with it. It’s important that the markup of a component remains separate from other components to ensure each can handle all of its concerns gracefully.
+
+##### Installing Yarn
```

Comment:
- [ ] Move to a new "Installation" page under "Getting Started"
- [ ] Use markdown instead of `<a />` tag for links